### ISSUE TITLE
Dial numbers within #

### ISSUE THREAD

**Issue by CoericK** [anarchicknight/react-native-communications#12]
I want to dial a number witch contains a #, like "#987654321" but it doest work, if i dial "987654321" it works

**Comment by anarchicknight** [comment]
I did some digging in to this. I can confirm that when the number had # in it phonically would not work on Android or iOS.

I have found that if you encode the string before passing it to the phonically method e.g. # is %23 then the phonically will work on Android. However, it still did not work on iOS.

Unfortunately it seems that Apple do not allow the use of certain characters in the tel: url scheme which is what this module uses. See the following links for information.

http://stackoverflow.com/questions/4660951/how-to-use-tel-with-star-asterisk-or-hash-pound-on-ios?lq=1

https://developer.apple.com/library/ios/featuredarticles/iPhoneURLScheme_Reference/PhoneLinks/PhoneLinks.html

In summary, the developer can work around this in Android by encoding the string, but it cannot work for iOS using the tel: url scheme methods. You would need to dig in to native code instead.

Sorry to be the bearer of bad news.
